documenting.rst

```diff
@@ -1565,11 +1565,12 @@ You can also use ``make help`` to see a list of targets supported by
 
 See also :file:`Doc/README.rst` for more information.
 
+Using sphinx-build
+------------------
 
-Without make
-------------
-
-Install the Sphinx, blurb, and python-docs-theme packages from PyPI.
+Sometimes we directly want to execute the sphinx-build tool, in this case, you can
+use the following command line but before you have to install the Sphinx, blurb,
```

Comment:
"command line from the ``Doc`` directory (make sure to install Sphinx, blurb, and python-docs-theme from PyPI)::" and link to their PyPI pages. This then means you should remove line 1575.